R/object-import.R

```diff
@@ -19,7 +19,12 @@ merge.rd_section_reexport <- function(x, y, ...) {
 format.rd_section_reexport <- function(x, ...) {
   pkgs <- split(x$value$fun, x$value$pkg)
   pkg_links <- map2(names(pkgs), pkgs, function(pkg, funs) {
-    links <- paste0("\\code{\\link[", pkg, "]{", escape(sort(funs)), "}}",
+    funs <- sort(funs)
+    files <- vapply(funs, find_topic_in_package_reexp, character(1), pkg = pkg)
+    links <- paste0(
+      "\\code{\\link[", pkg,
+      ifelse(files == funs, "", paste0(":", files)),
```

Comment:
I _think_ it be simpler for `find_topic_in_package_reexp()` to do the pasting together of package and file?

Reply:
Just tried, one part, yes. It makes the other part more cumbersome, as we cannot to `ifelse(files == funs, ...)` any more, would need to `ifelse(files == paste0(pkg, ":", funs), ...)` which is not terrible, but the `files == funs` part is/was quite expressive, i.e. we only do this if the file name is different than the topic name.

Also, all the other `find_topic_in_package*()` functions just return the topic, so I think we can just stick with that?

Reply:
Ok, sounds good.